SampleApplications/Samples/NetCoreConsoleClient/Program.cs

```diff
@@ -113,6 +117,25 @@ public static int Main(string[] args)
             Console.WriteLine("1 - Create an Application Configuration.");
             exitCode = ExitCode.ErrorCreateApplication;
 
+            CertificateIdentifier applicationCert;
+            if (Utils.IsRunningOnMono())
+            {
+                applicationCert = new CertificateIdentifier
+                    {
```

Comment:
Nag: Inconsistent indentation...

Reply:
Ctrl-K-F auto format ...

---
In reply to: [163231560](https://github.com/OPCFoundation/UA-.NETStandard/pull/325#discussion_r163231560) [](ancestors = 163231560)